src/xdocs/config_whitespace.xml

```diff
@@ -2976,6 +2980,18 @@ public void myTest() {
 
     try {} catch (Exception e){} // OK
     try{} catch (Exception e){} // violation ''try' is not followed by whitespace'
+
+    try {}
+    finally { System.out.println("Hello"); } // OK
```

Comment:
Remove content of finally block, make it empty.
Make try-finally as single line, same as you did with try-catch

Reply:
@romani 
Line:  
`*      try {} catch (Exception e){} finally{} // violation ''finally' is not followed by whitespace'` in WhiteSpaceAfterCheck.java has more than 100 length.

Should I adjust the indentation for all or should I just remove whitespace between `finally{} //violation` and `violation''finally'`
`*      try {} catch (Exception e){} finally{}// violation''finally' is not followed by whitespace'`

Reply:
I have changed `Exception --> Error` in the catch block as In WhitespaceAfterCheck.java,
line was crossing 100+ length

Reply:
Good job!